styles/ide-haskell-repl.less

```diff
@@ -45,7 +63,7 @@ div.ide-haskell-repl {
     overflow-y: scroll;
     pre {
       font-size: inherit;
-      font-family: inherit;
+      font-family: monospace;
```

Comment:
Output area font is the same as main editor font (see https://github.com/atom-haskell/ide-haskell-repl/blob/master/lib/ide-haskell-repl-view.coffee#L56). Main editor font is usually monospace. So I don't see how this thing would make anything better.